regression/cbmc/r_w_ok_inconsistent_dead/test-global-no-cp.desc

```diff
@@ -0,0 +1,10 @@
+FUTURE
+main.c
+--function test_global_pointer --pointer-check --no-simplify --no-propagation
```

Comment:
:question: Could we include somewhere an explanation why the `no-propagation` flag is needed to be tested separately?

Reply:
Do you think I should add an explanation to every .desc file? Basically we want to test with `--no-simplify --no-propagation` (which turns off expression simplification and constant propagation) and without those two, so as to check that both the constraint encoding and the simplification/constant propagation work correctly.